package.json

```diff
@@ -10,8 +10,9 @@
     "cardigan": "pushd cardigan && yarn dev",
     "catalogue": "pushd catalogue/webapp && yarn dev",
     "content": "pushd content/webapp && yarn dev",
-    "deployCatalogue": "pushd deploy && ./deploy_service.js -s catalogue",
-    "deployContent": "pushd deploy && ./deploy_service.js -s content"
+    "build-webapps": "echo '@weco/catalogue @weco/content @weco/identity @weco/identity-admin' | xargs -n 1 | xargs -I% yarn workspace % run build",
+    "clean": "rimraf '**/.next **/lib'",
```

Comment:
What's the `lib`?

Reply:
`identity` uses it but I've just realised I don't want the whole thing in the action...